cartography/driftdetect/driftdetector.py

```diff
@@ -0,0 +1,107 @@
+import json
```

Comment:
`model` is probably a better name for this module. It contains the data model used by drift detection and functions for interacting with the data.